doc/api/errors.md

```diff
@@ -647,144 +644,129 @@ urlSearchParams.has.call(buf, 'foo');
 <a id="ERR_INVALID_TUPLE"></a>
 ### ERR_INVALID_TUPLE
 
-An error with code `'ERR_INVALID_TUPLE'` is thrown when an element in the
-`iterable` provided to the [WHATWG][WHATWG URL API] [`URLSearchParams`
-constructor][`new URLSearchParams(iterable)`] does not represent a `[name,
-value]` tuple – that is, if an element is not iterable, or does not consist of
-exactly two elements.
+Used when an element in the `iterable` provided to the [WHATWG][WHATWG URL
+API] [`URLSearchParams`constructor][`new URLSearchParams(iterable)`] does not
```

Comment:
Should, but by the magic of GFM it works:
![image](https://user-images.githubusercontent.com/96947/28098948-53db9582-6687-11e7-80c0-bd861817549e.png)